modules/cluster/addons.tf

```diff
@@ -59,7 +59,11 @@ module "kube_proxy" {
   config = local.config
   manifest = templatefile(
     "${path.module}/addons/kube-proxy.yaml",
-    { aws_region = data.aws_region.current.name },
+    {
+      aws_region   = data.aws_region.current.name
+      bind_address = var.support_ipv6 ? "0.0.0.0" : "127.0.0.1"
```

Comment:
I am not sure the support_ipv6 == true part of this branch is appropriate...

As I understand the comment here https://github.com/aws/amazon-vpc-cni-k8s/issues/1078#issuecomment-804655593 the idea is that we can bypass some detection for v4 vs v6 if we specify 127.0.0.1...

Since I don't think we are using ipv6 stack at all anywhere I am not sure how to test the behaviour ... I think it would be fine to hardcode this, and make a note in the readme that we don't currently support ipv6